src/main/java/jdbc/JdbcTemplate.java

```diff
@@ -21,6 +21,18 @@ public void execute(final String sql) {
         }
     }
 
+    public void executeAndReturnGeneratedKeys(final String sql, final IdMapper idMapper) {
+        try (final Statement statement = connection.createStatement()) {
+            statement.execute(sql, Statement.RETURN_GENERATED_KEYS);
+            ResultSet generatedKeys = statement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                idMapper.mapRow(generatedKeys);
+            }
+        } catch (Exception e) {
+            throw new RuntimeException(e);
```

Comment:
```suggestion
            throw new RuntimeException("JdbcTemplate error Blabla: " + e.getMessage(), e);
```

에러메시지를 챙겨줘도 좋을것 같아서요~ 🤔

Reply:
기존 코드 복붙하느라 에러메시지는 생각도 못했었네요ㅎㅎ